maintenance_purchase/tests/test_purchase_order.py

```diff
@@ -0,0 +1,206 @@
+# Copyright 2024 ACSONE SA/NV
```

Comment:
Is this copyright correct? Don't you put as contributor?

Reply:
Thanks @pedrobaeza,
I am currently working for Acsone., so the copyright is correct 🙂